### DISCUSSION TITLE
Dont Work any game on snapdragon 845, android 10 (Samsung galaxy s9)

### DISCUSSION THREAD

**Discussion by UWolfff** [skyline-emu/skyline#1669] [Q&A]
I've tried to play Holow Knight, the binding of isaac, and Pokemon Diamond and none works (there are videos where these games run, and I tried old and new versions of skyline). I read that it may be because my device has outdated drivers, but I can't update them because I don't have root. Does anyone know a method to fix it? Or if the developers can, fix it with an update

**Comment by mateoalfaro** [comment]
Would be better if you asked for help on the discord since some of those should work

**Reply by UWolfff** [reply]
I already asked on discord, and they told me to update drivers. I also have an ugly visual bug in Celeste when I do dash. And I would like to update drivers, but for that I need root and the usb port of my cell phone is broken xD

**Reply by hope3x** [reply]
Your only option is to wait then.
